public/js/main.js

```diff
@@ -20,6 +20,8 @@ const state = {
   isInferenceOn: true,
   label: true,
   video: undefined,
+  // TODO Consider putting the pathname to a JS[ON]/YAML file which would also be read by app.js
+  devMode: window.location.pathname === '/',
```

Comment:
nit: rename to isInDevMode or similar (basically anything that starts with "is" prefix denoting that this is binary state variable)

Reply:
Good call, I'll do that.